apps/docs/app/tokens/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import Link from "next/link";
+import React from "react";
+import "./aside.css";
+
+export const Aside = () => {
```

Comment:
Can you make it dynamic and place it in `app/components/Aside/`?

Reply:
By dynamic you mean being able to auto detect all the H2 in the page?